text-serializer-legacy/src/main/java/net/kyori/adventure/text/serializer/legacy/LegacyComponentSerializer.java

```diff
@@ -103,7 +105,7 @@ public interface LegacyComponentSerializer extends ComponentSerializer<Component
    * @return the legacy format
    * @since 4.0.0
    */
-  static @Nullable LegacyFormat parseChar(final char character) {
```

Comment:
`@Nullable` should remain.

Reply:
did not mean to remove this one oops, will fix tomorrow.